packages/protocol/contracts/stability/SortedOracles.sol

```diff
@@ -215,16 +222,16 @@ contract SortedOracles is ISortedOracles, Ownable, Initializable {
    * @return The number of oracle report timestamps for `token`.
    */
   function numTimestamps(address token) external view returns (uint256) {
-    return timestamps[token].numElements;
+    return timestamps[token].list.list.numElements;
```

Comment:
It's a little unpleasant that consumers (like `SortedOracles`) of the libraries have to worry about this abstraction depth
Not immediately clear there is a workaround

Reply:
It should have used `getNumElements()`, done

Reply:
Don't see this change checked in

Reply:
Forgot to push, done